resources/i18n/cs_CZ/cura.po

```diff
@@ -1440,15 +1440,23 @@ msgid_plural ""
 "You are about to remove {0} printers from Cura. This action cannot be undone.\n"
 "Are you sure you want to continue?"
 msgstr[0] ""
+"Chystáte se odebrat {0} tiskárnu from Cury. Tuto akci nelze vrátit zpět.\n"
```

Comment:
"Chystáte se odebrat {0} tiskárnu z Cury. Tuto akci nelze vrátit zpět.\n"

Reply:
Oops. Fixed. Thank you for noticing. :+1: